.github/workflows/main.yml

```diff
@@ -114,4 +114,4 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: clippy
-          args: -- -D warnings
+          args: --all-targets -- -D warnings
```

Comment:
`--all-targets`'s "target" means lib, bin, example, test, bench, etc., not `--target`'s "target (target triple)".